_posts/2019-09-09-what-is-mtls.md

```diff
@@ -4,25 +4,86 @@ current: post
 
-## What can go wrong if you do not use mTLS?
-Not using mTLS may compromise your brand, your company, and your services in a multitude of ways. Consider a scenario where hackers compromise your device and use a commonly shared API key stored on disk to gain access to your back-end servers. This means that any sensitive information stored on your server might become accessible. Without mutual authentication there is no way to confirm client-side identity and isolate the compromised device. Imagine again, the occasion where without mTLS hackers take over a device and start sending falsified data. The data received and transmitted is altered, compromised and unreliable. But the users have no way of knowing this and they keep making critical decisions based on the compromised data received. They analyze metrics, they receive real-time data but…the integrity of the data is compromised so nothing can be trusted. What if the data compromised is related to health and smart healthcare devices? Or if it’s logistics-related data or even manufacturing and smart transportation data? What kind of a cataclysmic or disastrous impact will compromised data have? This is why in IoT infrastructure mTLS is an important piece of the security puzzle. IoT security is all about authenticating, trusting and securing the services provided.
+mTLS is just a type of TLS (Transport Layer Security). This is commonly found in verifying endpoints and for encrypting communications and also browsers. The 'https' in a url is a sign of a TLS controlled website. This part of the url verifies the identity of the server, the website you're accessing, to you, the client. 
+Unlike a password which is manually entered by a single user, TLS is typically managed by cryptographic certificates. Most TLS protocols usually verify this with an x509 certificate; and it's typically the client requesting a valid certificate from the server (DNS name on server must match that of the certificate). Secure Sockets Layer (SSL) security is based off this principle and https is its successor.
 
-## How can this affect you and your company?
-For companies, the financial implications of security breaches are tremendous and difficult to recover from. It is not only the brand’s reputation plummeting but it is also the loss of customers and most importantly the huge fines incurred. The brand’s reliability is questioned and trust is lost.
+The main thing that makes mTLS different (and arguably more secure) is that it requires *both* the server and client to verify each other: a handshake. This is particularly useful in instances where you want two servers to communicate with each other; or in a scenario where *you* are the server, not the client. Ultimately it's flexible and lightweight being perfect for edge devices.
```

Comment:
"want two servers to communicate with each other" - we need to be careful about the terminology here. By definition, two 'servers' (from the application perspective) cannot communicate with each other. One has to be a client.